### PR TITLE
Add Crystal support (based on Ruby)

### PR DESCRIPTION
**Pull request by phongnh**
Initial support for Crystal lang, please help to review/approve it, thanks.

image

### CODE REVIEW

File: autoload/ctrlp/funky/ft/crystal.vim [hunk 1]
```diff
@@ -0,0 +1,64 @@
+" Language: Crystal (crystal)
+" Author: phongnh <nhphong1406@gmail.com>
+" License: The MIT License
+
+let s:fu = ctrlp#funky#getutils()
```
Comment (tacahiroy) [inline autoload/ctrlp/funky/ft/crystal.vim:5]: This variable looks like not used in the script. Can you please remove this?

**Review by tacahiroy — COMMENTED** [review]
Thanks for the PR. Can you take a look the comment and apply the fix please.

File: autoload/ctrlp/funky/ft/crystal.vim (hunk 1 above)
Comment (phongnh) [reply autoload/ctrlp/funky/ft/crystal.vim]: Removed. Please help to re-check this, thanks.

**Review by tacahiroy — APPROVED** [review]
LGTM. Thanks for fixing that.

**Comment by tacahiroy** [comment]
Merged! Thank you!

**Pull request merged**